docs/user_guide/assessment.rst

```diff
@@ -397,6 +397,61 @@ see the :ref:`sphx_glr_auto_examples_plot_new_metrics.py` notebook in the
 :ref:`examples`.
 
 
+Plotting with MetricFrame
+-------------------------
+MetricFrame pd.DataFrame properties can be plotted using matplotlib. Here is an example,
+referencing the same `metric_c_f` object above.
+
+.. code-block::
+
+    metric_c_f.by_group.plot(kind="bar",
+                             title='Show accuracy_score metric')
+
+It is possible to customize the plots. Here are some common examples:
+
+Customize Plots: `ylim`
+^^^^^^^^^^^^^^^^^^^^^^^
+The yaxis range is automatically set, which sometimes can be misleading, hence it is
+sometimes useful to set the `ylim` argument to define the yaxis range.
+
+.. code-block::
+
+    metric_c_f.by_group.plot(kind="bar",
+                             ylim=[0,1],
+                             title='Show accuracy_score metric')
+
+Customize Plots: `kind`
+^^^^^^^^^^^^^^^^^^^^^^^
+There are different types of charts (e.g. pie, bar, line) which can be defined by the `kind`
+argument. Some metrics are better represented as a pie chart, some as a bar chart. Here
+is an example of a pie chart representing the `count` metric.
+
+.. code-block::
```

Comment:
This is excellent! However, when using `code-block` the code isn't actually run during the doc build. Instead, I suggest using the same technique I used for the "other" plotting doc section (in this same file towards the end) using `literalinclude` from the `plot_quickstart.py` file. I would suggest extending it a bit and adding these other plot representations (like pie), but that would also mean that you'd have to restrict the existing occurrences of `literalinclude` for that file, but I can point you there when we get to that point.

Reply:
Question: do we have specific cases in mind in which a pie chart is preferable over a bar chart? In the dataviz community pie charts are usually scrutinized quite a bit because they can be tricky to read (see e.g., [this blogpost](https://www.data-to-viz.com/caveat/pie.html): TDLR, people are bad at reading angles).

Reply:
Got it! Adjusted the documentation to use `literal include` and also amended the comment around the `kind` param (pie/bar chart) to be more generic - simply showing the reader of the option to switch up the various chart types